src/app/containers/CpsAsset/index.jsx

```diff
@@ -30,7 +28,7 @@ const StyledSectionLabel = styled(SectionLabel)`
   margin-top: 0;
 `;
 
-// Apply the right margin-top between the section label and the promos
+// Apply the right padding-top&bottom around the single story promo
```

Comment:
```suggestion
// Apply the correct top & bottom padding around the single story promo
```
Just struggled to understand this comment, I thought it was referring to `padding-right`, just a reword suggestion